docs/source/complex_numbers.rst

```diff
@@ -0,0 +1,174 @@
+can easily to switch using the complex tensors in their code using :func:`torch.view_as_complex` and
+- :func:`torch.view_as_real`:
+
+::
+
+     >>> x = torch.randn(3, 2)
+     >>> x
+     tensor([[ 0.6125, -0.1681],
+          [-0.3773,  1.3487],
+          [-0.0861, -0.7981]])
+     >>> y = torch.view_as_complex(x)
+     >>> y
+     tensor([ 0.6125-0.1681j, -0.3773+1.3487j, -0.0861-0.7981j])
+     >>> torch.view_as_real(y)
+     tensor([[ 0.6125, -0.1681],
+          [-0.3773,  1.3487],
+          [-0.0861, -0.7981]])
+
+Accessing real and imag
+-----------------------
+
+<1 line description>
+
+::
+
+     >>> y.real
+     tensor([ 0.6125, -0.3773, -0.0861])
+     >>> y.imag
+     tensor([-0.1681,  1.3487, -0.7981])
+
+Angle and abs
+-------------
+
+<1 line description>
+
+::
+
+     >>> x1=torch.tensor([3j, 4+4j])
+     >>> x1.abs()
+     tensor([3.0000, 5.6569])
+     >>> x1.angle()
+     tensor([1.5708, 0.7854])
+
+Serialization
+-------------
+
+<1 line description>
+
+::
+
+     >>> torch.save(y, 'complex_tensor.pt')
+     >>> torch.load('complex_tensor.pt')
+     tensor([ 0.6125-0.1681j, -0.3773+1.3487j, -0.0861-0.7981j])
+
+Autograd
+--------
+
+PyTorch supports Autograd for Complex Numbers.
+
+1. :func:`torch.functional.backward` can be used for holomorphic :math:`C -> C` functions.
+   For non-holomorphic functions, the gradient is evaluated as if it were holomorphic.
+2. :func:`torch.functional.backward` can be used to optimize :math:`C -> R` functions, like
+   real-values loss functions of complex parameters :math:`x` by taking steps in the direction
+   of conjugate of :math:`x.grad`.
+3. mention the current behavior of backward for spectral ops?
+
+For more details, check out the Autograd note. <add the doc link when PR is merged>
+
+> do we need this code or is it better to add it in tutorial?
+
+::
+
+     x = torch.randn(2, 2, 2, dtype=torch.double, requires_grad=True)>>> x = torch.randn(2, 2, 2, dtype=torch.double, requires_grad=True)
+     >>> y = x.detach().requires_grad_(True)
+     >>> x0 = x.clone()
+     >>> x1 = torch.view_as_complex(x0)
+     >>> x2 = torch.view_as_real(x1)
+     >>> x2.mul_(2)
+     tensor([[[ 4.2425, -0.1076],
+          [ 3.2731,  2.3156]],
+
+          [[ 4.1179,  0.7358],
+          [-1.7711, -0.4389]]], dtype=torch.float64,
+          grad_fn=<ViewAsRealBackward>)
+     >>> x2.sum().backward()
+     >>> y0 = y.clone()
+     >>> y0.mul_(2)
+     tensor([[[ 4.2425, -0.1076],
+          [ 3.2731,  2.3156]],
+
+          [[ 4.1179,  0.7358],
+          [-1.7711, -0.4389]]], dtype=torch.float64, grad_fn=<MulBackward0>)
+     >>> y0.sum().backward()
+     >>> x.grad
+     tensor([[[2., 2.],
+          [2., 2.]],
+
+          [[2., 2.],
+          [2., 2.]]], dtype=torch.float64)
+     >>> y.grad
+     tensor([[[2., 2.],
+          [2., 2.]],
+
+          [[2., 2.],
+          [2., 2.]]], dtype=torch.float64)
+
+We also do not support the following subsystems:
```

Comment:
"also" is a little strange here, as this doesn't follow the previous sections (which are things that are supported).

Reply:
yeah you are right updated!